### PR TITLE
Add -quiet flag to genpkey

### PR DESCRIPTION
**Pull request by richsalz**
Picking up late suggestions to PR #6909 by Philip Prindeville
<philipp@redfish-solutions.com>.

### CODE REVIEW

**Review by t8m — APPROVED** [review]

**Review by pprindeville — APPROVED** [review]
LGTM

**Comment by t8m** [comment]
Sanitizer CI failure is unrelated.

**Review by slontis — APPROVED** [review]

File: apps/genpkey.c [hunk 1]
```diff
@@ -332,16 +338,25 @@ static int genpkey_cb(EVP_PKEY_CTX *ctx)
...
     BIO *b = EVP_PKEY_CTX_get_app_data(ctx);
-    int p;
-    p = EVP_PKEY_CTX_get_keygen_info(ctx, 0);
-    if (p == 0)
+
+    if (quiet)
+        return 1;
+
+    switch (EVP_PKEY_CTX_get_keygen_info(ctx, 0)) {
+    default:
```
Comment (richsalz) [inline apps/genpkey.c:346]: I don't want to reset the clock, but this default case can be removed given the assignment at line 339.

Comment (pprindeville) [reply apps/genpkey.c:346]: It's your changeset...

Comment (slontis) [inline apps/genpkey.c:346]: just removing it from the assignment on 339 would also be fine.

**Comment by richsalz** [comment]
I pushed a fixup to remove the defualt case since it's handled by the initialization.  You folks decide to reset the clock or not. :)

**Review by t8m — APPROVED** [review]
I do not think this resets the clock. Merging.

**Comment by t8m** [comment]
Merged to master assuming @slontis is OK with this as well. Thank you.

**Pull request closed without merging**